docs/source/architecture/rest_api.rst

```diff
@@ -192,11 +193,18 @@ endpoint-specific queries are not listed here.
    * - **max**
      - For paging, specifies the id or index of the last resource to fetch. It
        would be used instead of *min*, not in the same query.
+   * - **sort**
+     - For endpoints that fetch lists of resources, specifies a key or keys to
+       sort the list by. Sorts can be modified with a simple but powerful
+       syntax: comma-separate multiple keys; dot-notate nested keys; use a
```

Comment:
Based on the previous sentence part, this sounds like it is describing the syntax, so using "comma-separated", "dot-notated" adjectives makes more sense. The second set of syntax items sounds like instructions on available operations. Suggest changing to something like the following (note the order about starting with keys, and working up to what can be done with them):

Sorts can be modified with the following rules:
1.  keys follow dot notation. For example, `header.signer_pubkey`
2. `header` may be omitted for nested header keys
3.  append `.length` to sort by length of a field
4. multiple keys may be comma-separated
5. Use of a minus sign to indicate descending order.  E.g. `-transaction_ids.length`